### PR TITLE
Add release note for 1.0.2 release

### PR DESCRIPTION
**Pull request by pubudu91**

### CODE REVIEW

File: release-notes-1.0.2.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Overview to Ballerina 1.0.2
```
Comment (MaryamZi) [inline release-notes-1.0.2.md]: Shouldn't this be 
```suggestion
# Overview of Ballerina 1.0.2
```

Noticed we have "to" in the previous release notes too, but shall we clarify?

File: release-notes-1.0.2.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+Ballerina 1.0.2 release is a continuation of the effort to stabilize and improve the 1.0 release while maintaining backward compatibility.
```
Comment (MaryamZi) [inline release-notes-1.0.2.md]:
```suggestion
The Ballerina 1.0.2 release is a continuation of the effort to stabilize and improve the 1.0.0 release while maintaining backward compatibility. 
```

File: release-notes-1.0.2.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Overview to Ballerina 1.0.2
+
+Ballerina 1.0.2 release is a continuation of the effort to stabilize and improve the 1.0 release while maintaining backward compatibility. 
+This release also focuses on improving the performance of the compiler, build tools and developer tools.
```
Comment (MaryamZi) [inline release-notes-1.0.2.md]:
```suggestion
This release also focuses on improving the performance of the compiler, build tools, and developer tools.
```

File: release-notes-1.0.2.md (hunk 1 above)
Comment (pubudu91) [reply release-notes-1.0.2.md]: of sounds correct though. Let's go with that.

**Pull request merged**